_includes/navbar.html

```diff
@@ -1,102 +1,236 @@
-          <ul class="dropdown-menu">
-            <li><a href="/">Ontology Table</a></li>
-            <li><a href="/id-policy.html">ID Policy</a></li>
-            <li><a href="http://dashboard.obofoundry.org/dashboard/index.html">OBO Dashboard</a></li>
-            <li role="separator" class="divider"></li>
-            <li><a href="https://github.com/OBOFoundry/OBOFoundry.github.io/tree/master/ontology">Ontologies Metadata Source</a></li>
-            <li><a href="https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/registry/">Combined Metadata</a></li>
-            <li><a href="https://raw.githubusercontent.com/OBOFoundry/OBOFoundry.github.io/master/registry/ontologies.yml">Ontology YAML</a></li>
-            <li><a href="https://raw.githubusercontent.com/OBOFoundry/OBOFoundry.github.io/master/registry/ontologies.jsonld">Ontology JSON-LD <i>alpha</i></a></li>
-            <li><a href="https://raw.githubusercontent.com/OBOFoundry/OBOFoundry.github.io/master/registry/ontologies.ttl">Ontology RDF/Turtle <i>alpha</i></a></li>
-          </ul>
-        </li>
-        
-        <li>
-          <a href="/resources">Resources</a>
-        </li>
-        
-        <li class="dropdown">
-          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Citation <span class="caret"></span></a>
-          <ul class="dropdown-menu">
-            <li><a href="/docs/Citation.html">Cite Ontologies</a></li>
-            <li><a href="/registry/publications.html">Cite the OBO Foundry</a></li>
-            <li><a href="/registry/publications.html">Publications Related to the OBO Foundry</a></li>
-          </ul>
-        </li>
+      <li>
+        <a href="/resources">Resources</a>
+      </li>
 
-        <li class="dropdown">
-          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Participate <span class="caret"></span></a>
-          <ul class="dropdown-menu">
-            <li><a href="https://github.com/OBOFoundry/OBOFoundry.github.io/issues">OBO Foundry Issue Tracker</a></li>
-            <li><a href="https://groups.google.com/forum/#!forum/obo-discuss">OBO-Discuss Mail List</a></li>
-            <li><a href="https://groups.google.com/forum/#!members/obo-tools">OBO-Tools Mail List</a></li>
-            <li><a href="https://obo-communitygroup.slack.com/archives/C01DP18L5GW">OBO Community Slack channel</a></li>
-             <li><a href="/faq/how-do-i-register-my-ontology.html">Submit your ontology</a></li>
-          </ul>
-        </li>
+      <li class="dropdown">
+        <a
+          href="#"
+          class="dropdown-toggle"
+          data-toggle="dropdown"
+          role="button"
+          aria-haspopup="true"
+          aria-expanded="false"
+          >Citation <span class="caret"></span
+        ></a>
```

Comment:
Is this how the new linters like it? Looks a bit odd, do you know the motivation for it?

Reply:
I guess so. We don't have to argue with what the linter does, assuming it's deterministic, even if we don't really like it ourselves.

Reply:
No problem, I agree! Just curious.